src/main/java/org/jabref/logic/exporter/TemplateExporter.java

```diff
@@ -181,14 +143,13 @@ private Reader getReader(String filename) throws IOException {
     }
 
     @Override
-    public void performExport(final BibDatabaseContext databaseContext, final String file,
-            final Charset encoding, List<BibEntry> entries) throws Exception {
+    public void export(final BibDatabaseContext databaseContext, final Path file,
+                       final Charset encoding, List<BibEntry> entries) throws Exception {
         Objects.requireNonNull(databaseContext);
         Objects.requireNonNull(entries);
         if (entries.isEmpty()) { // Do not export if no entries to export -- avoids exports with only template text
             return;
         }
-        Path outFile = Paths.get(file);
         SaveSession ss = null;
```

Comment:
Again, since you are already refactoring this method, please also improve the naming of variables.